src/pretix/base/forms/auth.py

```diff
@@ -3,6 +3,9 @@
 from django.utils.translation import ugettext_lazy as _
 
 from pretix.base.models import User
+from django.contrib.auth.password_validation import (
+    validate_password, password_validators_help_texts
+)
```

Comment:
We use ``isort`` to keep imports deterministically ordered. Just run ``isort -rc pretix`` on the codebase and those will be ordered alphabetically.